src/App.tsx

```diff
@@ -1,19 +1,180 @@
 import React from 'react';
-import './App.css';
+import classNames from 'classnames';
+import styles from './App.scss';
+import { TodoList } from './components/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+const cx = classNames.bind(styles);
+
+interface State {
+  visualisedTodos: any,
+  userName: string,
+  todoTitle: string,
+  isDisabled: boolean,
+  errorTitle: boolean,
+  errorUser: boolean,
+}
+
+interface Todo {
+  userId: number,
+  id: number,
+  title: string,
+  completed: boolean,
+  user: any,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    visualisedTodos: [],
+    userName: '',
+    todoTitle: '',
+    isDisabled: false,
+    errorTitle: true,
+    errorUser: true,
+  };
+
+  componentDidMount() {
+    const updateTodos = todos.map(todo => (
+      {
+        ...todo,
+        user: users.find(user => user.id === todo.userId),
+      }
+    ));
+
+    this.setState({
+      visualisedTodos: updateTodos,
+    });
+  }
+
+  setUser = (event: any) => {
```

Comment:
```suggestion
  setUser = (event) => {
```
Не используй any, в реакте есть тип для этого ивента